CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_CXX_STANDARD 17)
 set(OpenGL_GL_PREFERENCE GLVND)
 
+add_subdirectory(src)
```

Comment:
What's the output layout in this case?

Should we specify `binary_dir` so we can separate product output from test/samples output?

Reply:
Its the default at this moment in time, so currently its `out/build/x64-Debug/samples` for this target. It copies from the specified `bin` folder of the lib project output for dynamic linkage.

I think as a result the samples program(s) is/are already separated?

Reply:
Yep, that looks to be the case. Sounds good to me.